Client/src/Pages/Auth/Register/StepThree/index.jsx

```diff
@@ -0,0 +1,161 @@
+	return (
+		<>
+			<Stack
+				gap={{ xs: theme.spacing(8), sm: theme.spacing(12) }}
+				textAlign="center"
+			>
+				<Box>
+					<Typography component="h1">Sign Up</Typography>
+					<Typography>Create your password</Typography>
+				</Box>
+				<Box
+					component="form"
+					noValidate
+					spellCheck={false}
+					onSubmit={onSubmit}
+					textAlign="left"
+					sx={{
+						display: "grid",
+						gap: theme.spacing(8),
+						"& .input-error": {
+							display: "none",
+						},
+					}}
+				>
+					<Field
+						type="password"
+						id="register-password-input"
+						name="password"
+						label="Password"
+						isRequired={true}
+						placeholder="Create a password"
+						autoComplete="current-password"
+						value={form.password}
+						onChange={handleChange}
+						error={errors.password && errors.password[0]}
+						ref={inputRef}
+					/>
+					<Field
+						type="password"
+						id="register-confirm-input"
+						name="confirm"
+						label="Confirm password"
+						isRequired={true}
+						placeholder="Confirm your password"
+						autoComplete="current-password"
+						value={form.confirm}
+						onChange={handleChange}
+						error={errors.confirm && errors.confirm[0]}
+					/>
+					<Stack
+						gap={theme.spacing(4)}
+						mb={{ xs: theme.spacing(6), sm: theme.spacing(8) }}
+					>
+						<Check
+							noHighlightText={"Must be at least"}
+							text={"8 characters long"}
+							variant={feedbacks.length}
+						/>
+						<Check
+							noHighlightText={"Must contain at least"}
+							text={"one special character"}
+							variant={feedbacks.special}
+						/>
+						<Check
+							noHighlightText={"Must contain at least"}
+							text={"one number"}
+							variant={feedbacks.number}
+						/>
+						<Check
+							noHighlightText={"Must contain at least"}
+							text={"one upper character"}
+							variant={feedbacks.uppercase}
+						/>
+						<Check
+							noHighlightText={"Must contain at least"}
+							text={"one lower character"}
+							variant={feedbacks.lowercase}
+						/>
+						<Check
+							noHighlightText={"Confirm password and password"}
+							text={"must match"}
+							variant={feedbacks.confirm}
+						/>
+					</Stack>
+					<Stack
+						direction="row"
+						justifyContent="space-between"
+					>
+						<Button
+							variant="outlined"
+							color="info"
+							onClick={onBack}
+							sx={{
+								px: theme.spacing(5),
+								"& svg.MuiSvgIcon-root": {
+									mr: theme.spacing(3),
+								},
+								":focus-visible": {
+									outline: `2px solid ${theme.palette.primary.dark}`,
+									outlineOffset: "4px",
+								},
+							}}
+							props={{ tabIndex: -1 }}
+						>
+							<ArrowBackRoundedIcon />
+							Back
+						</Button>
+						<Button
+							type="submit"
+							variant="contained"
+							color="primary"
+							disabled={
+								form.password.length === 0 ||
+								form.confirm.length === 0 ||
+								Object.keys(errors).length !== 0
```

Comment:
Appreciate the strict evalutations here, not a big fan of JavaScripts type coercion